### ISSUE TITLE
Cannot build docker image

### ISSUE THREAD

**Issue by yorek** [rickbergfalk/sqlpad#711]
Docker image cannot be built as the following error pops up:

```
api/batches
    ✓ Creates batch
    1) GETs finished result


  3 passing (11s)
  1 failing

  1) api/batches
       GETs finished result:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/sqlpad/server/test/api/batches.js)
      at listOnTimeout (internal/timers.js:549:17)
      at processTimers (internal/timers.js:492:7)
```

and after that, the build process stops.

I'm using Windows 10 WSL 2 (32GB, 8vCores)

**Comment by rickbergfalk** [comment]
@yorek how are you building the image?

I'm on Windows 10 with WSL 2 as well, but things work for me. I run docker commands using cmder, from Windows as opposed to from bash inside WSL 2 if that makes sense? I switched back to Windows a few months ago and I'm still sometimes confused about what is running where.

**Comment by rickbergfalk** [comment]
@yorek I wonder if something is running into an error and it isn't obvious because the logs are set to silent? 

I've pushed changes to `server/test/utils.js` to log error level messages. Hopefully next test run we'll find something pointing to the issue.

**Comment by yorek** [comment]
Will do. I tried both running the test using WSL or Powershell....and it doesn't make any difference. This is expected as when Docker starts everything happens inside the Docker container so no matter how you start it, the result should be the same

**Comment by yorek** [comment]
Btw I'm building the image using a very simple
```
docker build . -t sqlpad
```

**Comment by yorek** [comment]
Ok, after synching by fork with the upstream repo I'm now able to build the image. I also fixed an issue with the view I changed.
Unfortunately now I cannot push my changes due to #719

**Comment by yorek** [comment]
Fixed (see #719)
